exonum-light-client/src/test/java/com/exonum/client/ExplorerApiHelperTest.java

```diff
@@ -30,21 +30,53 @@
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.is;
 import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
 
+import com.exonum.binding.common.blockchain.ExecutionStatuses;
 import com.exonum.binding.common.blockchain.TransactionLocation;
-import com.exonum.binding.common.blockchain.TransactionResult;
 import com.exonum.binding.common.hash.HashCode;
 import com.exonum.binding.common.message.TransactionMessage;
 import com.exonum.client.response.Block;
 import com.exonum.client.response.BlockResponse;
 import com.exonum.client.response.BlocksResponse;
 import com.exonum.client.response.TransactionResponse;
 import com.exonum.client.response.TransactionStatus;
+import com.exonum.core.messages.Runtime.ErrorKind;
+import com.exonum.core.messages.Runtime.ExecutionStatus;
 import java.time.ZonedDateTime;
+import java.util.stream.Stream;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
 
 class ExplorerApiHelperTest {
 
+  private static TransactionMessage TRANSACTION_MESSAGE = createTransactionMessage();
+  private static long BLOCK_HEIGHT = 1L;
+  private static long INDEX_IN_BLOCK = 0L;
+
+  private static String TEMPLATE_TRANSACTION_MESSAGE_JSON = "{\n"
+      + "    'type': 'committed',\n"
+      + "    'content': '" + toHex(TRANSACTION_MESSAGE) + "',\n"
+      + "    'location': {\n"
+      + "        'block_height': " + BLOCK_HEIGHT + ",\n"
+      + "        'position_in_block': " + INDEX_IN_BLOCK + "\n"
+      + "    },\n"
+      + "    'location_proof': {\n"
```

Comment:
I took this `location_proof` from some core tests. It looks weird, but we don't use it in any way, so as long as it's a valid json, we are good.